src/terra-dev-site/test/workspace/MockContent.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import { useActiveMainPage } from '../../../application-container';
+import NotificationBanner from '../../../notification-banner/NotificationBanner';
+import styles from './TestStyles.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  initialCount: PropTypes.number,
+  title: PropTypes.string,
+  onShowActivityOverlay: PropTypes.func,
+  onShowStatusOverlay: PropTypes.func,
+};
+
+const MockContent = ({
+  initialCount = 0, title = '', onShowActivityOverlay, onShowStatusOverlay,
+}) => {
+  const activeMainPage = useActiveMainPage();
+
+  const [clickCount, setClickCount] = useState(initialCount);
+  const [showAlertBanner, setShowAlertBanner] = useState(false);
+  const [showWarningBanner, setShowWarningBanner] = useState(false);
+  const [showErrorBanner, setShowErrorBanner] = useState(false);
+
+  React.useEffect(() => {
+    setTimeout(() => {
+      setShowAlertBanner(true);
+    }, 5000);
+    setTimeout(() => {
+      setShowWarningBanner(true);
+    }, 8000);
+    setTimeout(() => {
+      setShowErrorBanner(true);
+    }, 11000);
+
+    setTimeout(() => {
+      setShowAlertBanner(false);
+    }, 12000);
+    setTimeout(() => {
+      setShowWarningBanner(false);
+    }, 13000);
+    setTimeout(() => {
+      setShowErrorBanner(false);
+    }, 14000);
+  }, []);
+
+  // React.useEffect(() => {
+  //   setTimeout(() => {
+  //     setShowAlertBanner(true);
+  //   }, 5000);
+  //   setTimeout(() => {
+  //     setShowWarningBanner(true);
+  //   }, 5000);
+  //   setTimeout(() => {
+  //     setShowErrorBanner(true);
+  //   }, 5000);
+
+  //   setTimeout(() => {
+  //     setShowAlertBanner(false);
+  //   }, 10000);
+  //   setTimeout(() => {
+  //     setShowWarningBanner(false);
+  //   }, 10000);
+  //   setTimeout(() => {
+  //     setShowErrorBanner(false);
+  //   }, 10000);
+  // }, []);
```

Comment:
```suggestion
```

Reply:
I'll get this when we've wrapped up accessibility testing in its entirety